templates/page.html

```diff
@@ -432,10 +432,11 @@ <h3>Near-term</h3>
         <div class="card">
           <div class="card-body">
             <h5 class="card-title">
-              <a href="https://github.com/orgs/reanahub/projects/50" class="card-link">Workspace file retention</a>
+              <a href="https://github.com/reanahub/reana-client-go" class="card-link">Go client and stable API</a>
```

Comment:
Should I remove the link or point to something else?

Reply:
We usually link to Kanban boards there, but we don't have one yet... I guess we can keep the link to the Go client repository.